app/src/main/java/org/dharmaseed/android/TalkRepository.java

```diff
@@ -77,16 +79,11 @@ private Cursor getTalks(
             innerJoin += joinDownloadedTalks();
         }
 
+        innerJoin += joinTalkIdVenueId();
+
         String where = "";
         if (searchTerms != null && searchTerms.size() > 0)
         {
-            // we only need the center table if we have search terms
```

Comment:
We need the center table all the time now.

Reply:
only if the caller passes in a column containing the `center` table name. There should be a check like I've done with `teachers`.